plantseg/run_plantseg.py

```diff
@@ -15,6 +15,8 @@ def parser():
 
 
 def main():
+    from plantseg.utils import check_version
+    check_version()
```

Comment:
would be good to surround this with `try-except` clause in case there is a problem with github/network. We should not fail if external service is down

Reply:
Fixed it